### ISSUE TITLE
Lighthouse audit flagged insufficient colour contrast on the 'Submit' button

### ISSUE THREAD

**Issue by bobbysebolao** [FAC-Sixteen/Wonder3-V2#11]
You scored 61 overall on the Lighthouse accessibility audit. When you get a mo, you might want to add ARIA labels to the input and submit buttons. Also, I can't tell if the submit button is focused on when I tab through on my keyboard?
